docs/pages/web/datepicker.js

```diff
@@ -140,9 +139,20 @@ export default function DatePickerPage({ generatedDocGen }: {| generatedDocGen:
       <AccessibilitySection name={generatedDocGen?.displayName} />
 
       <MainSection name="Variants">
-        <MainSection.Subsection columns={2} title="Controlled component">
+        <MainSection.Subsection
+          columns={2}
+          title="Controlled component"
+          description={`
+DatePicker is a controlled component. Use \`value\`, \`onChange\`, \`onClearInput\` and \`onError\` to implement it correctly.
+
+DatePicker is controlled when \`value\` is not "undefined". When \`value\` is "undefined", it stays uncontrolled.
+        `}
+        >
           <MainSection.Card
             title="Empty input"
+            description={`
+If DatePicker doesn't present pre-selected date values, initialize \`value\` with "null" so the component is controlled.
+`}
             sandpackExample={
               <SandpackExample
```

Comment:
Can we hide the code controls on this example or add a header inside the example?

Reply:
1. code is shown w=in variants
2. we never hide controls
3. This is out of scope, I'll let you decide in the docs comprehension improvement if we should bring to the top the titles of the examples. Otherwise I'm breking consistency